qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -303,6 +304,33 @@
     def_rzz.append(inst, qargs, cargs)
 _sel.add_equivalence(RZZGate(theta), def_rzz)
 
+# RZXGate
+
+q = QuantumRegister(2, 'q')
+theta = Parameter('theta')
+def_rzx = QuantumCircuit(q)
+for inst, qargs, cargs in [
+        (HGate(), [q[1]], []),
+        (CXGate(), [q[0], q[1]], []),
+        (RZGate(theta), [q[1]], []),
+        (CXGate(), [q[0], q[1]], []),
+        (HGate(), [q[1]], [])
+]:
+    def_rzx.append(inst, qargs, cargs)
+_sel.add_equivalence(RZXGate(theta), def_rzx)
+
+# ECRGate
+
+q = QuantumRegister(2, 'q')
+def_ecr = QuantumCircuit(q)
+for inst, qargs, cargs in [
+        (RZXGate(pi/4), [q[0], q[1]], []),
+        (XGate(), [q[0]], []),
+        (RZXGate(-pi/4), [q[0], q[1]], [])
+]:
+    def_ecr.append(inst, qargs, cargs)
+_sel.add_equivalence(ECRGate(), def_ecr)
+
```

Comment:
Can you add an equivalence in terms of CX and 1q gates?

Reply:
is it needed? it will lower it via the RZX route and optimize, at least at level 3 (added a test for that)

Reply:
Not essential, and only for levels 0 and 1, it might save two two qubit gates (depending on what the target basis is).